samples/custom_dispatch/cpu/plugin/README.md

```diff
@@ -92,12 +92,26 @@ target_include_directories(...)
 
 ```mlir
 func.func private @simple_mul_workgroup(
-    %binding0: memref<?xf32>, %binding1: memref<?xf32>, %binding2: memref<?xf32>,
+    %binding0_baseptr: memref<f32>, %binding0_offset: index,
+    %binding1_baseptr: memref<f32>, %binding2_offset: index,
+    %binding2_baseptr: memref<f32>, %binding2_offset: index,
     %dim: index, %tid: index)
 ...
-func.call @simple_mul_workgroup(%memref0, %memref1, %memref2, %dim, %tid) : (memref<?xf32>, memref<?xf32>, memref<?xf32>, index, index) -> ()
+func.call @simple_mul_workgroup(
+    %memref0_baseptr, %memref0_offset,
+    %memref1_baseptr, %memref1_offset,
+    %memref2_baseptr, %memref2_offset,
+    %dim, %tid)
+    : (memref<f32>, index, memref<f32>, index, memref<f32>, index, index, index) -> ()
 ```
 
+For each of the two inputs and one output, a `<baseptr, offset>` pair is used
+to get the position to read from. It is essential for the implementations of
+these functions to manually perform the `baseptr + offset` before reading the
+data. The `memref` semantics in MLIR only guarantee that the `baseptr + offset`
+represents the valid position to read from. Also note that the `offset` here
+is in number of elements (i.e. number of floats).
+
```

Comment:
indent so that this stays indented with the listing:
![image](https://user-images.githubusercontent.com/75337/235502472-bab5510e-cdb2-41bd-be7a-c6aab5fa8bd9.png)

(maybe something like:)
```suggestion
   For each of the two inputs and one output, a `<baseptr, offset>` pair is used
   to get the position to read from. It is essential for the implementations of
   these functions to manually perform the `baseptr + offset` before reading the
   data. The `memref` semantics in MLIR only guarantee that the `baseptr + offset`
   represents the valid position to read from. Also note that the `offset` here
   is in number of elements (i.e. number of floats).

```